test/unit/controller.js

```diff
@@ -578,7 +578,7 @@ exports['Tessel.get'] = {
         test.equal(this.closeTesselConnections.callCount, 1);
         test.equal(Array.isArray(this.closeTesselConnections.args[0]), true);
         test.equal(this.logsInfo.callCount, 2);
-        test.equal(_.contains(this.logsInfo.lastCall.args[0], 'the_name'), true);
+        test.equal(this.logsInfo.lastCall.args[0].includes('the_name'), true);
```

Comment:
```text
➜  test  node --version
v5.5.0
➜  test  node 
> [1, 2, 3].includes(2)
TypeError: [1,2,3].includes is not a function
    at repl:1:11
    at REPLServer.defaultEval (repl.js:252:27)
    at bound (domain.js:287:14)
    at REPLServer.runBound [as eval] (domain.js:300:12)
    at REPLServer.<anonymous> (repl.js:417:12)
    at emitOne (events.js:95:20)
    at REPLServer.emit (events.js:182:7)
    at REPLServer.Interface._onLine (readline.js:211:10)
    at REPLServer.Interface._line (readline.js:550:8)
    at REPLServer.Interface._ttyWrite (readline.js:827:14)
```

Am I missing something, I didn't think that `includes` was landed in Node.js yet, I think it's only stage 4.

[`array-includes`](https://www.npmjs.com/package/array-includes) is a nice replacement :+1: